spec/requests/api/merge_requests_spec.rb

```diff
@@ -358,14 +361,27 @@
 
   describe "POST /projects/:id/merge_request/:merge_request_id/comments" do
     it "should return comment" do
-      post api("/projects/#{project.id}/merge_request/#{merge_request.id}/comments", user), note: "My comment"
-      expect(response.status).to eq(201)
-      expect(json_response['note']).to eq('My comment')
+      post api(merge_request_comments_url, user), note: 'My comment'
+      response.status.should == 201
+      json_response['note'].should == 'My comment'
+    end
+
+    it 'should return comment with line and file_path' do
+      post api(merge_request_comments_url, user),
+           note: 'comment with line and path',
+           line: 5,
+           file_path: 'files/ruby/feature.rb',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.